### ISSUE TITLE
Activating the production mode questions

### ISSUE THREAD

**Issue by koldirkaev** [csob/paymentgateway#116]
Hello! We have tested gateway with test keys and test cards and all was OK. But after successful MID initialization, generating and changing keys, we have error 400. I have some questions. How long it takes to public key sent to bank to start work? It is possible to generate keys again?

**Comment by ratajskym** [comment]
Hi, did you confirm public key in Posmerchant web application. In test environment is your public key confirmed automatically but in production you must do it by your self using OTP that was send to your technical email.

**Comment by koldirkaev** [comment]
Thank you for help. Yes. We finally confirmed keys in Posmerchant/Payment Gateways section. But problem still remains. Now we receive "payment/init failed, http response: 404" error. The key question is what URL should we use in setup.php? I tried https://api.platebnibrana.csob.cz/api/v1.5/ and just https://api.platebnibrana.csob.cz/ I include our setup.php screenshot in a case if it is something missed in it.
![2016-01-15 20 37 54](https://cloud.githubusercontent.com/assets/15154487/12360333/633e09d0-bbc8-11e5-889c-5441339fc821.png)
![2016-01-15 20 39 07](https://cloud.githubusercontent.com/assets/15154487/12360453/295f1f82-bbc9-11e5-8ffa-b9ac315452a2.png)
